system-tests/tests/smoke/cre/por_test.go

```diff
@@ -580,6 +691,43 @@ func setupPoRTestEnvironment(
 		dfConfigErr := configureDataFeedsCacheContract(testLogger, dfConfigInput)
 		require.NoError(t, dfConfigErr, "failed to configure data feeds cache")
 
+		var rci *readContractInput
+		if rcti != nil {
```

Comment:
could we do that workflow registration outside of setup function? that way we would avoid this if as well and wouldn't have to pass `readContractInput` to this function at all.